python/pyspark/sql/tests.py

```diff
@@ -30,6 +30,19 @@
 import functools
 import time
 import datetime
+import traceback
+
+if sys.version_info[:2] <= (2, 6):
```

Comment:
can we follow the existing style here? You can send a PR to update all of them later.